### DISCUSSION TITLE
V5 Ignore directories on node ace build?

### DISCUSSION THREAD

**Discussion by KararTY** [adonisjs/core#1350] [Help] [answered]
Is there a way to ignore directories on `node ace build`?

I tried adding the directory to `.adonisrc.json`'s `metaFiles[]` as such: `"resources/js/**"` but after build when I check `build/resources/js` I see that they've been built too, as well as copied over (I expected the copy-over, as that's what metaFiles' supposed to do).

I'm using a bundler for the front-end Javascript that I am writing in Typescript, and as such I want the Typescript files to just copy-over and not get compiled into Javascript by `node ace build`.

**Comment by thetutlage** [comment, marked as answer]
How about using `tsconfig` to ignore these files? Just add it to the `exclude` array.

```json
{
  "exclude": [
    "resources/**/*",
    "node_modules",
    "build"
  ],
}
```

**Reply by KararTY** [reply]
Of course, perfect. Thank you.
